frontend/src/layout/navigation-3000/components/TopBar.tsx

```diff
@@ -88,17 +88,13 @@ export function TopBar(): JSX.Element | null {
                 <div className="TopBar3000__breadcrumbs">
                     {breadcrumbs.length > 1 && (
                         <div className="TopBar3000__trail">
-                            {breadcrumbs.slice(0, -1).map((breadcrumb, index) => (
+                            {breadcrumbs.slice(0, -1).map((breadcrumb) => (
                                 <React.Fragment key={breadcrumb.name || '…'}>
-                                    <Breadcrumb breadcrumb={breadcrumb} index={index} />
+                                    <Breadcrumb breadcrumb={breadcrumb} />
```

Comment:
nit: Almost feels like we have a more reliable keying mechanism here for the fragment as well that it's weird we don't use it.

Reply:
Low value - just wondering if it could save us from a confusing future bug where two breadcrumbs have the same name

Reply:
For sure, I missed this